types/office-js-preview/index.d.ts

```diff
@@ -17730,6 +17729,628 @@ declare namespace Excel {
+         * @beta
+         */
+        percentComplete?: number;
+        /**
+         *
+         * Sets a new priority for the task. This is a value between 0 and 10, where 0 represents the highest priority.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        priority?: number;
+        /**
+         *
+         * Sets if the change should remove all previous assignees from the task. When this property is set to true `emailsToUnassign` property will have no effect and emails specified in `emailsToAssign` property will become the only assignees.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        removeAllPreviousAssignees?: boolean;
+        /**
+         *
+         * Sets a new the start date for the task, in UTC time zone. Can be set to `null` to remove the start date. Should be set together with `dueDate` to avoid conflicts.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        startDate?: Date;
+        /**
+         *
+         * Sets a new title for the task.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        title?: string;
+    }
+    /**
+     *
+     * Represents a collection of tasks.
+     *
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    class TaskCollection extends OfficeExtension.ClientObject {
+        /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
+        context: RequestContext;
+        /** Gets the loaded child items in this collection. */
+        readonly items: Excel.Task[];
+        /**
+         * Gets the number of tasks in the collection.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        getCount(): OfficeExtension.ClientResult<number>;
+        /**
+         * Gets a task using its id.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param key The id of the task.
+         * @returns The task with the given id. If there is no task with the given id, then an error is thrown.
+         */
+        getItem(key: string): Excel.Task;
+        /**
+         * Gets a task by its index in the collection.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param index The index of the task in the collection.
+         * @returns The task with the given index.
+         */
+        getItemAt(index: number): Excel.Task;
+        /**
+         * Gets a task using its id.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param key The id of the task.
+         * @returns The task with the given id. If there is no task with the given id, a null object is returned.
```

Comment:
Maybe this can wait till Alison does it for all of Excel, but we should replace "a null object is returned" with "an object is returned whose `isNullObject` property is set to true"

Reply:
I submitted the *orNullObject metadata updates to the engineer a few weeks ago, so they should be in the pipeline! I was hoping they'd be in this update :(